workshop/exercises/html/leaflet.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
Please rename this file to something more meaningful than "leaflet.hml" (look at the other example on this folder)